cmd/quill/cli/commands/describe.go

```diff
@@ -42,26 +42,26 @@ func Describe(app clio.Application) *cobra.Command {
 			},
 		),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return app.Run(cmd.Context(), async(func() error {
-				var err error
-				buf := &strings.Builder{}
-				switch strings.ToLower(opts.Output) {
-				case "text":
-					err = extract.ShowText(opts.Path, buf, !opts.Detail)
-				case "json":
-					err = extract.ShowJSON(opts.Path, buf)
-				default:
-					err = fmt.Errorf("unknown format: %s", opts.Output)
-				}
+			defer bus.Exit()
```

Comment:
Should this be part of the CLIO `app.Setup` wrapper?

Reply:
exiting the bus is a calling application concern. This exit capability was rolled up into the helper when it existing in quill, but couldn't get ported over. Could be something we make better in the future.